curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/616d50b93ba424d6282c99cf.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-12
 
 # --description--
 
-Notice that your marker doesn't seem to have any color. The background color was actually applied, but since the marker `div` element is empty, it doesn't have any height by default.
+The background color was applied, but since the marker `div` element is empty, it doesn't have any height by default.
```

Comment:
I took out the `Notice that your marker doesn't seem to have any color.` Do you think I should add this back? I only took it out because in the previous step we say, `Note: You will not see any changes after adding the CSS`

Reply:
I think that's fine, the user should now understand why it is not showing any changes. Adding a notice on a subject that was already in place may seem redundant.